plugins/janus_videoroom.c

```diff
@@ -3564,10 +3706,55 @@ static void *janus_videoroom_handler(void *data) {
 					b = (int)(videoroom->bitrate/1000);
 				char sdp[1280], audio_mline[256], video_mline[512], data_mline[256];
 				if(audio) {
-					g_snprintf(audio_mline, 256, sdp_a_template,
-						opus_pt,						/* Opus payload type */
-						audio_mode,						/* The publisher gets a recvonly or inactive back */
-						opus_pt); 						/* Opus payload type */
+					switch (videoroom->acodec) {
```

Comment:
I am so sorry. I don't use IDE, my custom editor ruined it. I will be more careful from now on.